### ISSUE TITLE
Textarea content is white on white in dark mode

### ISSUE THREAD

**Issue by ryansechrest** [TryGhost/Casper#923]
`input` fields have a darker color:

```
html.dark-mode .gh-content input {
    color: #303a3e;
}
```

But it doesn't seem to target `textarea`, which causes white on white text.

**Comment by ryansechrest** [comment]
Something I noticed is that when Casper is manually switched from light to dark, a `dark-mode` class is available so that we can target the `textarea` with a CSS injection on the page to temporarily fix this, however if the theme is in auto mode (and it's currently dark), there's a `auto-color` class.

Is there a way the light/dark mode can be targeted with a CSS code injection when it's in this auto mode?

**Comment by minimaluminium** [comment]
Hey @ryansechrest, the auto dark mode can be targeted with a CSS media query like below.

```css
@media (prefers-color-scheme: dark) {
    html.auto-color ... {

    }
}
```

This is now fixed in 5.4.7 though. Thanks for the report :)
